projects/core/src/store-finder/facade/store-finder.service.ts

```diff
@@ -14,22 +14,40 @@ export class StoreFinderService {
 
   constructor(private store: Store<StoresState>, private winRef: WindowRef) {}
 
-  getStoresLoading(): Observable<any> {
+  /**
+   * Returns boolean observable for store's loading state
+   */
+  getStoresLoading(): Observable<boolean> {
     return this.store.pipe(select(fromStore.getStoresLoading));
   }
 
+  /**
+   * Returns observable for store's entities
+   */
   getFindStoresEntities(): Observable<any> {
```

Comment:
replace `any` with `FindStoreEntity[]` or something similar. note: you'll have to create this type first

Reply:
Added special interface for entities array.